ExposureApp/CRExposureNotification/Views/Loader/Loader.swift

```diff
@@ -28,7 +28,7 @@ class Loader: UIView {
     func loadViewFromNib() -> UIView {
         let bundle = Bundle(for: Loader.self)
         let nib = UINib(nibName: className, bundle: bundle)
-        let view = nib.instantiate(withOwner: self, options: nil)[0] as! UIView
+        let view = nib.instantiate(withOwner: self, options: nil).first as! UIView
```

Comment:
Maybe a guard would be good here too ?
`guard let view = nib.instantiate(withOwner: self, options: nil).first as? UIView else { return UIView() }`
or anything else that would gracefully handle the event of it not being able to be cast as UIView.

Reply:
Maybe even insted of returning UIView() to do:
fatalError(..)

Reply:
Done :)